src/node_crypto.cc

```diff
@@ -3684,8 +3684,8 @@ void Hmac::HmacInit(const FunctionCallbackInfo<Value>& args) {
 bool Hmac::HmacUpdate(const char* data, int len) {
   if (!initialised_)
     return false;
-  HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
-  return true;
+  int r = HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
+  return r == 1;
```

Comment:
Note: This should result in a `ThrowTypeError("HmacUpdate fail")` under some conditions, but I haven't found any test coverage for that. Still looking.